doc/classloader.md

```diff
@@ -41,4 +41,9 @@ It's also possible to augument `GroovyClassLoader` classpath via
 This is more suitable for scripts that are considered a part of the
 system, such as global libraries or plugin code.
 
-
+Those scripts are not snapshotted & persisted with the running program,
+so if they change while the program is running, with or without Jenkins restarts
+in the middle, then the program could fail. (For example, if a program
+call stack includes a class from a global library and that class goes away,
+then the program fails to survive the restart because the call stack cannot
```

Comment:
I consider that a part of another ticket where we support locking in on a specific commit of a global library. In the documentation I'm merely describing the current behaviour.

I don't consider it a "serious flaw" - the same problem happens if you upgrade plugins that incompatibly change the behaviours, etc.